packages/@lightningjs/ui-components/src/components/Base/Base.d.ts

```diff
@@ -18,6 +18,8 @@
 
 import lng from '@lightningjs/core';
 
+type BaseStyle = object;
```

Comment:
this works, but we might be able to use a more explicit type here 

> If you want a type meaning "empty object", you probably want Record<string, never> instead.
> If you want a type meaning "any object", you probably want object instead.
> If you want a type meaning "any value", you probably want unknown instead.
^ https://stackoverflow.com/a/75753960

I think `Record<string, never>` is the most semantically correct, but I also think it'll break the way we do style types. so object might be fine. just a thought